xbmc/cores/VideoPlayer/VideoRenderers/LinuxRendererGLES.cpp

```diff
@@ -268,26 +276,13 @@ void CLinuxRendererGLES::LoadPlane(CYuvPlane& plane, int type,
 
   if (stride != static_cast<int>(width * bps))
   {
-#if HAS_GLES >= 3
-    unsigned int verMajor, verMinor;
-    m_renderSystem->GetRenderVersion(verMajor, verMinor);
-
-    if (verMajor >= 3)
+    if (m_unpackRowLengthSupported)
     {
       glGetIntegerv(GL_UNPACK_ROW_LENGTH, &pixelStore);
       glPixelStorei(GL_UNPACK_ROW_LENGTH, stride);
       pixelStoreKey = GL_UNPACK_ROW_LENGTH;
     }
     else
-#elif defined (GL_UNPACK_ROW_LENGTH_EXT)
-    if (m_renderSystem->IsExtSupported("GL_EXT_unpack_subimage"))
```

Comment:
This extension should be exposed if your library reports GLES 2.0 but supports the required property.

See, https://www.khronos.org/registry/OpenGL/extensions/EXT/EXT_unpack_subimage.txt